### DISCUSSION TITLE
Filament Demo Page is Broken - Main Body Doesn't Load

### DISCUSSION THREAD

**Discussion by ahajicek07** [filamentphp/filament#9032] [General]
![image](https://github.com/filamentphp/filament/assets/51858995/021cff65-f464-4aea-85ff-378538516995)

**Comment by sinnbeck** [comment]
I had the exact same thing happen. It suddenly started working. Not sure why. Perhaps give a reboot a try

https://github.com/filamentphp/filament/discussions/9001
